### DISCUSSION TITLE
Provide review comments on binary files

### DISCUSSION THREAD

**Discussion by rdecarreau** [community/community#28085] [Pull Requests]
My team and I use LabVIEW very extensively. LabVIEW represents source code files as binary files; setting aside feelings about that, it would be nice to have the same PR "Conversation" workflow available on a binary file. I should be able to simply select a binary file by name from the file tree, and start a Conversation. Right now I can only do that on a line within a viewable file type. I am relegated to making basic comments with numbers or checkboxes and reviewees cannot reply to or close specific comments. It's very confusing.

**Comment by YounGCoder09** [comment]
Why are you still using LabVIEW ?

**Comment by ljfitz** [comment]
The same request applies to sub-modules and `git lfs` files. I've no answer for this, but I found this request as I was looking for an open issue on sub-modules and `git lfs` files, and I suppose they would also fall under this issue. (I see I have to click `Answer`, but I'm aware this is not an answer.)

**Comment by rdecarreau** [comment]
How can we get some traction on this?

**Comment by bewuethr** [comment]
This is on the roadmap at github/roadmap#536.
